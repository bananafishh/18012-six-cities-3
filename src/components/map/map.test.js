import React from 'react';
import renderer from 'react-test-renderer';

import Map from './map';

const coords = [
  [52.3909553943508, 4.85309666406198],
  [52.369553943508, 4.85309666406198],
  [52.3909553943508, 4.929309666406198],
  [52.3809553943508, 4.939309666406198],
];

it(`Компонент «Map» рендерится корректно`, () => {
  const tree = renderer
    .create(
        <Map center={coords[0]} coords={coords}/>,
        {createNodeMock: () => document.createElement(`div`)}
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
