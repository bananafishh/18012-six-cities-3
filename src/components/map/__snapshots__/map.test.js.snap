// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Компонент «Map» рендерится корректно 1`] = `
<div
  style={
    Object {
      "height": "100%",
    }
  }
/>
`;
